gradle/libs.versions.toml

```diff
@@ -1,6 +1,7 @@
 [versions]
 opentelemetry = "1.37.0"
-opentelemetry-alpha = "2.2.0-alpha"
+opentelemetry-inst = "2.3.0"
```

Comment:
I think it's clearer with `inst` in the name, to know when you have the instrumentation version (vs. core version).

Reply:
I like the idea of replacing `alpha` with something more meaningful. I would prefer to avoid abbreviations though, I know `opentelemetry-instrumentation` could be a lot, though the first time I saw `inst`, for some reason "instrumentation" wasn't the first word that came to my mind 😅

Reply:
Ok, fine, I'll spell out `inst` -> `instrumentation`...but I'm not going to change semconv. :)